src/backend/utils.ts

```diff
@@ -969,14 +1050,47 @@ export async function downloadDefaultWine() {
   return null
 }
 
+export async function setGPTKDefaultOnMacOS() {
+  const isGPTKCompatible = await isMacSonomaOrHigher()
+  if (!isGPTKCompatible) {
+    return
+  }
+
+  const wineList = await GlobalConfig.get().getAlternativeWine()
+  const gptk = wineList.find((wine) => wine.type === 'toolkit')
+  if (!gptk) {
+    await downloadDefaultWine()
+    return
```

Comment:
how are we retrying after we early return here

Reply:
Good catch, I will call this method itself after finishing downloading Default Wine.